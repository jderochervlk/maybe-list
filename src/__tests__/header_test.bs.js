// Generated by ReScript, PLEASE EDIT WITH CARE
'use strict';

var Jest = require("@glennsl/rescript-jest/src/jest.bs.js");
var React = require("react");
var Header = require("../Header.bs.js");
var ReactTestingLibrary = require("../external/ReactTestingLibrary.bs.js");

Jest.test("displays text", (function (param) {
        return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.render(undefined, undefined, undefined, undefined, undefined, React.createElement(Header.make, {})).container));
      }));

/*  Not a pure module */
