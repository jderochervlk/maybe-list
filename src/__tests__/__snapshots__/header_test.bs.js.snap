// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays text 1`] = `
<div>
  <div
    class="css-1qkjgjc"
  >
    <p
      class="chakra-text css-1olzugq"
    >
      Maybe List
    </p>
    <p
      class="chakra-text css-542elh"
    >
      A todo app that deletes items if they haven't been completed in 24 hours.
    </p>
  </div>
</div>
`;
